metadata-ingestion/src/datahub/sql_parsing/sqlglot_utils.py

```diff
@@ -131,10 +131,16 @@ def _expression_to_string(
     # Replace anything that looks like a string with a placeholder.
     re.compile(r"'[^']*'"): "?",
     # Replace sequences of IN/VALUES with a single placeholder.
-    re.compile(r"\b(IN|VALUES)\s*\(\?(?:, \?)*\)", re.IGNORECASE): r"\1 (?)",
+    # The r" ?" makes it more robust to uneven spacing.
+    re.compile(r"\b(IN|VALUES)\s*\( ?\?(?:, ?\?)* ?\)", re.IGNORECASE): r"\1 (?)",
```

Comment:
Is there a specific reason you are not using \s which matches on tab as well?
```suggestion
    re.compile(r"\b(IN|VALUES)\s*\(\s?\?(?:,\s?\?)*\s?\)", re.IGNORECASE): r"\1 (?)",
```

Reply:
a previous regex rule replaces all runs of whitespace with a single space, so the single character is sufficient here